CHANGES.md

```diff
@@ -0,0 +1,1499 @@
+# OWASP ModSecurity Core Rule Set (CRS) CHANGES
+
+## Report Bugs/Issues to GitHub Issues Tracker or the mailinglist
+
+* <https://github.com/coreruleset/coreruleset/issues>
+  or the CRS Google Group at
+* <https://groups.google.com/a/owasp.org/forum/#!forum/modsecurity-core-rule-set-project>
+
+## Version 3.3.5 - 2023-07-18
+
+Important changes:
+
+* Backport fix for CVE-2023-38199 in new rule 920620
+
+Fixes:
+
+* PL related issues in rule 921422
+* Move auditLogPart if it is in a chained rules where needed
+
+Chore:
+
+* Cleaning up PL1 tags
+* Move testing framework from ftw to go-ftw
+* Backport yaml test files fixes to support go-ftw
```

Comment:
For the draft blog post announcement, I went through the associated PRs to be able to add proper attributions and to write slightly more detailed descriptions. We could copy-paste those descriptions into this proposed PR.

Or not, I don't mind :smile:

```suggestion
Important changes:

* Backport fix for CVE-2023-38199 from CRS v4 via new rule 920620 (Andrea Menin, Felipe Zipitría)

Fixes:

* Fix paranoia level-related scoring issue in rule 921422 (Walter Hop)
* Move auditLogParts actions to the end of chained rules where used (Ervin Hegedus)

Chore:

* Clean up redundant paranoia level tags (Ervin Hegedus)
* Clean up YAML test files to support go-ftw testing framework (Felipe Zipitría)
* Move testing framework from ftw to go-ftw (Felipe Zipitría)
```